lib/rake_tasks/schema_to_dbml.rake

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+namespace :schema_to_dbml do
```

Comment:
This rake task is not very useful yet but we can keep it in order to prepare the future